### PR TITLE
win11dragdroptaskbarfix: add version 2.2.0.0

### LINKED ISSUE #7490: [Request] Windows 11 Drag & Drop to the Taskbar (Fix)
### Information

**Name:** Windows 11 Drag & Drop to the Taskbar (Fix)

**Description:** This program fixes the missing "Drag & Drop to the Taskbar" support in Windows 11.

**Homepage:** https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix

**Download link(s):** https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/releases/download/v.2.0.0.0-release/Windows11DragAndDropToTaskbarFix.exe

**Some indication of popularity/repute:** GitHub stars 555

### PR DESCRIPTION
**Pull request by tech189**
- Closes #7490
- First try at using persist, is this the simplest way? The program creates a config file but only if you press "Configure..." in its tray menu. So I had to create it in pre_install, hope that's right.
- Also this is the shortest I could make the manifest name without leaving it unreadable/unsearchable.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/win11dragdroptaskbarfix.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ],
+    "shortcuts": [
+        [
+            "Windows11DragAndDropToTaskbarFix.exe",
+            "Windows 11 Drag & Drop to the Taskbar (Fix)"
+        ]
+    ],
+    "persist": [
+        "Windows11DragAndDropToTaskbarFixConfig.txt"
+    ],
```
Comment (rashil2000) [inline bucket/win11dragdroptaskbarfix.json:26]:
```suggestion
    "persist": "Windows11DragAndDropToTaskbarFixConfig.txt",
```

Is the application able to read this file correctly (when you create it in pre_install)?

File: bucket/win11dragdroptaskbarfix.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "2.2.0.0",
+    "description": "Fixes the missing 'Drag & Drop to the Taskbar' support in Windows 11.",
+    "homepage": "https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix",
+    "license": "GPL-3.0",
```
Comment (rashil2000) [inline bucket/win11dragdroptaskbarfix.json:5]:
```suggestion
    "license": "GPL-3.0-only",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/win11dragdroptaskbarfix.json (hunk 1 above)
Comment (tech189) [reply bucket/win11dragdroptaskbarfix.json:26]: Yes, for example if I change `StartThisProgramAsAdministrator=0` to `1`, the program opens as admin on first run.

**Pull request merged**